easybuild/easyconfigs/g/GenomeTester4/GenomeTester4-0.4-intel-2018a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'GenomeTester4'
+version = '0.4'
+
+homepage = 'https://github.com/bioinfo-ut/GenomeTester4'
+description = "A toolkit for performing set operations - union, intersection and complement - on k-mer lists."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/bioinfo-ut/GenomeTester4/archive/']
+sources = ['Version_4_0.tar.gz']
```

Comment:
@verdurin Hmm, I overlooked this earlier... How does this relate to the `0.4` version exactly? Should we use `%(version_minor)s` here?

Reply:
@boegel yes, it appears I made that up in attempt to reconstruct a release history that doesn't really exist.

If you look at https://github.com/bioinfo-ut/GenomeTester4/releases it's described as `test`.

Calling it version `4` feels wrong, nevertheless.

Reply:
I'd say `4.0` is the right version, and then you can use:

```python
sources = ['Version_%(version_major)s_%(version_minor)s.tar.gz']